### PR TITLE
ZBUG-1975 Portuguese, Date format showing wrong

### PR DESCRIPTION
**Pull request by subrahmanyaShetty090**
-changed the date format from mm/dd/yyyy to dd/mm/yyyy in com_zimbra_date_pt_BR.properties file .

### CODE REVIEW

File: src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-# This program is free software: you can redistribute it and/or modify it under
-# the terms of the GNU General Public License as published by the Free Software Foundation,
-# version 2 of the License.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-# See the GNU General Public License for more details.
-# You should have received a copy of the GNU General Public License along with this program.
-# If not, see <https://www.gnu.org/licenses/>.
+# Zimbra Collaboration Suite Zimlets
```
Comment (silentsakky) [inline src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties]: this looks changed, but couldn't notice any difference
please revert this change

File: src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties [hunk 2]
```diff
@@ -195,7 +195,7 @@ format17.pattern = {datenum} {monthname}.? {yearnum}
 format17.rule = now year={yearnum},{monthname},{datenum}
 
 # e.g. 9/23/1970
```
Comment (silentsakky) [inline src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties]: update comment

**Review by BhavinBathani — APPROVED** [review]

File: src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties (hunk 2 above)
Comment (subrahmanyaShetty090) [reply src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties]: updated the comment.

File: src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties (hunk 1 above)
Comment (subrahmanyaShetty090) [reply src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties]: Reverted these changes.

**Review by silentsakky — APPROVED** [review]
Please squash commits
